### ISSUE TITLE
Apple silicon installation has soundfile error

### ISSUE THREAD

**Issue by sammlapp** [kitzeslab/opensoundscape#532]
After installing opensoundscape without ffmpeg on my Apple Silicon M1 Pro chip laptop, I got an error about libsndfile not being found during `import opensoundscape` in python. 

```
OSError: cannot load library '/Users/SML161/miniconda3/envs/opso070/lib/python3.8/site-packages/_soundfile_data/libsndfile.dylib': dlopen(/Users/SML161/miniconda3/envs/opso070/lib/python3.8/site-packages/_soundfile_data/libsndfile.dylib, 0x0002): tried: '/Users/SML161/miniconda3/envs/opso070/lib/python3.8/site-packages/_soundfile_data/libsndfile.dylib' (no such file).  Additionally, ctypes.util.find_library() did not manage to locate a library called '/Users/SML161/miniconda3/envs/opso070/lib/python3.8/site-packages/_soundfile_data/libsndfile.dylib'
```

**Comment by sammlapp** [comment]
This is related to open issues in python-soundfile
https://github.com/bastibe/python-soundfile/issues/310 
https://github.com/ohmtech-rdi/eurorack-blocks/issues/444#issuecomment-1153155080

**Comment by sammlapp** [comment]
The workaround posted [here](https://github.com/ohmtech-rdi/eurorack-blocks/issues/444#issuecomment-1153155080) works for me and can be used until there is a fix. It involves copying the libsnfile site-packages files to the location where Python is (incorrectly) looking for them.

**Comment by sammlapp** [comment]
This is resolved as of the OpenSoundscape v0.7.1 release when using a Python 3.9 environment.
